src/Illuminate/Routing/ResourceRegistrar.php

```diff
@@ -137,24 +149,34 @@ public function getResourceUri($resource)
 		// paths however they need to, as some do not have any wildcards at all.
 		$segments = explode('.', $resource);
 
-		$uri = $this->getNestedResourceUri($segments);
+		$uri = $this->getNestedResourceUri($segments, $options);
 
-		return str_replace('/{'.$this->getResourceWildcard(last($segments)).'}', '', $uri);
+		$resource = last($segments);
+
+		if (isset($options['wildcards'][$resource]))
```

Comment:
I followed how it is done other places in the file such as:
```
if (isset($options['names'][$method])) return $options['names'][$method];
```
I would assume performance would be better keeping it to isset too.